stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/application/App.kt

```diff
@@ -1,40 +1,24 @@
 package io.getstream.chat.sample.application
 
 import android.app.Application
-import io.getstream.chat.sample.BuildConfig
-import io.getstream.chat.sample.data.dataModule
-import io.getstream.chat.sample.feature.custom_login.customLoginModule
-import io.getstream.chat.sample.feature.user_login.userLoginModule
-import org.koin.android.ext.koin.androidContext
-import org.koin.android.ext.koin.androidLogger
-import org.koin.core.context.startKoin
+import io.getstream.chat.sample.data.user.UserRepository
 
 class App : Application() {
 
+    // done for simplicity, a DI framework should be used in the real app
+    val chatInitializer = ChatInitializer(this)
+    var userRepository = UserRepository()
```

Comment:
This could be a `val`, right?
```suggestion
    val userRepository = UserRepository()
```